### PR TITLE
Add deepchem

### PR DESCRIPTION
**Pull request by hadim**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hadim** [comment]
Need https://github.com/conda-forge/staged-recipes/pull/10176 to be merged first.

**Comment by hadim** [comment]
deepchem depends on tensorflow and I don't know what is the best practice in conda-forge about this dep.

I personally never install tensorflow from conda but always from pip because new version comes faster, they are optimized by Google and I sometimes want to use tf-nightly too.

So I would be in favor of not depending on tensorflow for this recipe but I also understand the issues that it raises...

**Comment by hadim** [comment]
https://github.com/conda-forge/openmm-feedstock/pull/7 is a blocker. So we wait until it's solved.

**Comment by simonbray** [comment]
@hadim, it seems that pdbfixer is only used at one place in the deepchem code: https://github.com/deepchem/deepchem/blob/6c1ded50c94547352ba0d00ba991025886622322/deepchem/utils/rdkit_util.py#L33.

I don't think it affects the vast majority of deepchem functionality and if a user does need this function they can still easily install pdbfixer themselves from the omnia channel. So maybe just remove it from the dependencies list in this recipe?

File: recipes/deepchem/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - setuptools
+    - pbr
+    - numpy
+  run:
+    - python >=3.5
+    - pdbfixer
```
Comment (simonbray) [inline recipes/deepchem/meta.yaml]:
```suggestion
```

**Comment by simonbray** [comment]
Hi @hadim, I made a few small changes to the recipe which get it to build successfully on my machine. Could you merge please? Here is the link to the PR: https://github.com/hadim/staged-recipes/pull/1

**Comment by hadim** [comment]
Done.

When CI is ready you should ping someone from the python conda forge team so they can review and merge.

File: recipes/deepchem/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepchem/deepchem/archive/{{ version }}.tar.gz
+  sha256: eec0d4dfa23e234d26d171ef20c62a699052a1bc230bbace341047e0c2673030
+
+build:
+  noarch: python
+  number: 0
+  script: "PBR_VERSION=${PKG_VERSION} && export PBR_VERSION && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```
Comment (simonbray) [inline recipes/deepchem/meta.yaml]:
```suggestion
  script: "PBR_VERSION=${PKG_VERSION} && export PBR_VERSION && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
  skip: True   # [win]
```

Comment (simonbray) [reply recipes/deepchem/meta.yaml]: It seems xgboost is not available for windows, could you also commit this line @hadim?

File: recipes/deepchem/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (simonbray) [inline recipes/deepchem/meta.yaml]:
```suggestion
skip: True   # [win]

```

**Comment by simonbray** [comment]
@conda-forge/core, could someone review this?

@rbharath, maybe you could also cast an eye over this PR? We can also add you to the maintainer list if you like.

**Comment by hadim** [comment]
Could you also remove me as a maintainer actually, please?

**Comment by simonbray** [comment]
Actually I cannot edit the files in this PR, can you remove yourself?

**Comment by hadim** [comment]
@simonbray maybe it makes more sense to close this PR and you open your own? I am not really using deepchem anymore.

**Comment by simonbray** [comment]
Moved to https://github.com/conda-forge/staged-recipes/pull/10818.

**Comment by hadim** [comment]
Thanks @simonbray

**Pull request closed without merging**